src/index.jsx

```diff
@@ -8,6 +8,7 @@ import AdminDashboard from './pages/AdminDashboard/AdminDashboard';
 import AdminProfilePage from './pages/AdminProfilePage/AdminProfilePage';
 import ExportCSV from './pages/ExportCSV/ExportCSV';
 import AddBoxFormPage from './pages/AddBoxForm/AddBoxFormPage';
+import EmailForm from './components/Email/EmailForm';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "./components/Email/EmailForm"